concourse/pipelines/deploy.yml

```diff
@@ -291,9 +309,27 @@ jobs:
               -auto-approve
 
             terraform output -json > "$root_dir/govuk-terraform-outputs/govuk-terraform-outputs.json"
+
+            for app in content-store; do
```

Comment:
I may have missed something but where is `content-store` defined?

Reply:
Heh, this is really gross code 😆  Shellcheck was telling me off for it to.

`content-store` isn't a variable here, it's the literal string `"content-store"`. The loop is there because so that when we do the next app we can just add it like:

```bash
for app in content-store fronted; do
```

... perhaps that's premature optimisation and I should have started with `app="content-store"` and dropped the loop.

Ideally I would have used an array here, but the shell is `dash`, not `bash`, so there are no arrays.

Reply:
sorry, I got confused by it being without quotes. I always wondered why they make life so difficult and do not provide bash, does it make the container that much bigger?

Reply:
We're piggy backing on [GOV.UK PaaS's images](https://github.com/alphagov/paas-docker-cloudfoundry-tools/blob/main/terraform/Dockerfile).

We get `dash` because they chose to do `FROM alpine`, which tries really hard to keep itself small. `dash` instead of `bash` is just one part of that, but it is definitely annoying. It's also annoying when you do `fly hijack`, because by default that tries to use `bash`, which then errors because it's not in the container. You have to explicitly tell it to use `sh`.

At some point we'll want to own our own concourse task images, at which point we might decide to use bash as the shell instead...

On the other hand, if I'd had bash here I probably would have done some even more confusing stuff with arrays, so 🤷🏻  maybe it's a good thing?

Reply:
I've changed the for loop to a function in 0c3c0dd